databank/management/commands/sources/FDRS.py

```diff
@@ -25,7 +25,7 @@
     ("KPI_expenditureLC_CHF", CO.expenditures),
     ("KPI_PeopleVol_Tot", CO.volunteers),
     ("KPI_TrainFA_Tot", CO.trained_in_first_aid),
-    ("KPI_noLocalUnits", CO.branches),
+    ("KPI_noBranches", CO.branches),
```

Comment:
> Issue comment: https://github.com/IFRCGo/go-web-app/issues/1396#issuecomment-2388892005
> So, it has been just updated (KPI_noBranches) in the Data2 FDRS API (note that we only started collecting this information for 2023)

This needs to fetch from different endpoint?

Maybe this? https://data-api.ifrc.org/api/data2?apiKey=XX-YY-ZZ&KPI_Don_Code=DAF001&indicator=KPI_noBranches

Reply:
This is already fixed.